SoftLayer/managers/monitor.py

```diff
@@ -0,0 +1,85 @@
+"""
+    SoftLayer.monitor
+    ~~~~~~~~~~~~~~~~~~
+    Monitoring Manager/helpers
+
+    :license: MIT, see LICENSE for more details.
+"""
+
+
+from SoftLayer import utils
+
+
+class MonitoringManager(utils.IdentifierMixin, object):
+    """Manages monitoring for Hardware and Virtual Servers
+
+    :param SoftLayer.API.CLient client: an API client instance
+    :param string server_tpe: should be either
+                  'Hardware_Server' or 'Virtual_Guest'
+    """
+
+    def __init__(self, client, server_type='Hardware_Server'):
```

Comment:
A good point. My thought at the moment is to have Hardware/VirtualGuestMonitorManager extend MonitoringManager and have them call the parent init. And then move out the list_X_status to each manager. I think that might be a bit more elegant all while trying to avoid duplicating code as much as possible.